docs/jobs/jobfile.md

```diff
@@ -0,0 +1,153 @@
+## Job Definition File
+
+Job Definition File (JDF) a way how to submit a complex pipeline into a HyperQueue.
+It is a [TOML](https://toml.io/) file that describes tasks of a job.
+JDF provides all functionalities as command line interface of HyperQueue and also adds access to additional features: 
+
+* *Heterogeneous tasks* -- Job may be composed of different tasks
+* *Dependencies* --  Tasks may have dependencies
+* *Resource request alternatives* -- Task may have alternative resource requests, e.g.: 4 cpus OR 1 cpus and 1 gpu
+
+Note that these features are also available through Python interface.
+
+
+## Minimal example
+
+First, we create file with the following content:
+
+```toml
+[[task]]
+command = ["sleep", "1"]
+```
+
+Let us assume that we have named this file as ``myjob``,
```

Comment:
I would name it with the `.toml` extension. Also, the name is not the same as in the command (`myjob` vs `myfile`).